Fix sizing of path to contrib image (#331)
AlexsLemonade/OpenScPCA-analysis: docs/getting-started/making-your-first-analysis-contribution.md

# Making your first analysis contribution

> <figure markdown="span">
> 
>       ![Path from joining to making a contribution- Planning](../img/first-analysis-contribution-planning.png){width="550"}
> 
> </figure>
> 

Reviewer: ```suggestion
<!-- These images intentionally do not use HTML formatting --> 

![Path from joining to making a contribution- Planning](../img/first-analysis-contribution-planning.png){width="550"}
```

> <figure markdown="span">

Reviewer: ```suggestion
```

>       ![Path from joining to making a contribution- Implementation and Review](../img/first-analysis-contribution-implementation-review.png){width="700"}
>       
> </figure>

Reviewer: ```suggestion
![Path from joining to making a contribution- Implementation and Review](../img/first-analysis-contribution-implementation-review.png){width="700"}
```
